ui/src/components/remote_component/RemoteLoaderComponent.js

```diff
@@ -17,9 +17,10 @@ const LoadDynamicScript = ({ url, setReady, setFailed }) => {
 };
 
 // Dynamic Script Loading component wrapper
-export const ExperimentEngineLoaderComponent = ({
+export const RemoteLoaderComponent = ({
```

Comment:
Nit: I think it's ok to keep this component as it was, except may be make the `componentName` configurable. After all, the std ensembler UI is tied to the experiment engine and it's the engine whose configs we are loading.

If it helps, we can rename it from `ExperimentEngineLoaderComponent` -> `ExperimentEngineComponentLoader` :D

Reply:
Wait so do you mean we should keep both components separate, i.e. `StandardEnsemblerLoaderComponent` and `ExperimentEngineLoaderComponent`, or combining them as @terryyylim had suggested? 😅

Reply:
I just noticed the older comment. I do think it makes sense to combine the way you have it. But I don't think this should be called a generic `RemoteLoaderComponent` because the logic inside is still looking at getting some url, configs (which are coupled to the experiment engine). So, the implementation can remain as is, but I think it still makes sense to call it something related to experiment engine - hence my suggestion is to use `ExperimentEngineComponentLoader` as the name.

Reply:
Ok! I've just renamed the component as you have suggested!